tfhe/examples/utilities/generates_test_keys.rs

```diff
@@ -107,39 +110,39 @@ fn client_server_keys() {
         generate_boolean_keys(&BOOLEAN_PARAMS);
     } else {
         const PBS_KEYS: [ClassicPBSParameters; 13] = [
-            PARAM_MESSAGE_1_CARRY_1_KS_PBS,
-            PARAM_MESSAGE_1_CARRY_2_KS_PBS,
-            PARAM_MESSAGE_1_CARRY_3_KS_PBS,
-            PARAM_MESSAGE_1_CARRY_4_KS_PBS,
-            PARAM_MESSAGE_1_CARRY_5_KS_PBS,
-            PARAM_MESSAGE_1_CARRY_6_KS_PBS,
-            PARAM_MESSAGE_2_CARRY_1_KS_PBS,
-            PARAM_MESSAGE_2_CARRY_2_KS_PBS,
-            PARAM_MESSAGE_2_CARRY_3_KS_PBS,
-            PARAM_MESSAGE_3_CARRY_1_KS_PBS,
-            PARAM_MESSAGE_3_CARRY_2_KS_PBS,
-            PARAM_MESSAGE_3_CARRY_3_KS_PBS,
-            PARAM_MESSAGE_4_CARRY_4_KS_PBS,
+            PARAM_MESSAGE_1_CARRY_1_KS_PBS_GAUSSIAN_2M64,
+            PARAM_MESSAGE_1_CARRY_2_KS_PBS_GAUSSIAN_2M64,
+            PARAM_MESSAGE_1_CARRY_3_KS_PBS_GAUSSIAN_2M64,
+            PARAM_MESSAGE_1_CARRY_4_KS_PBS_GAUSSIAN_2M64,
+            PARAM_MESSAGE_1_CARRY_5_KS_PBS_GAUSSIAN_2M64,
+            PARAM_MESSAGE_1_CARRY_6_KS_PBS_GAUSSIAN_2M64,
+            PARAM_MESSAGE_2_CARRY_1_KS_PBS_GAUSSIAN_2M64,
+            PARAM_MESSAGE_2_CARRY_2_KS_PBS_TUNIFORM_2M64,
```

Comment:
we need both a TUniform and Gaussian 2_2 generation, I would move the TUniform abvoe and have a comment to separate the two